test/Index/index_global_actors.swift

```diff
@@ -0,0 +1,35 @@
+// RUN: %target-swift-ide-test -print-indexed-symbols -include-locals -source-filename %s | %FileCheck %s
+// REQUIRES: concurrency
+
+@globalActor
+actor CustomActor {
+  static let shared = CustomActor()
+}
+
+// Closure attributes
+
+func f() {
+  _ = { @MainActor in }
+  // CHECK:      [[@LINE-1]]:10 | class/Swift | MainActor | s:ScM | Ref,RelCont | rel: 1
+  // CHECK-NEXT: RelCont | function/Swift | f() | s:14swift_ide_test1fyyF
+  _ = { @CustomActor in }
+  // CHECK:      [[@LINE-1]]:10 | class/Swift | CustomActor | s:14swift_ide_test11CustomActorC | Ref,RelCont | rel: 1
+  // CHECK-NEXT: RelCont | function/Swift | f() | s:14swift_ide_test1fyyF
+}
+
+// Function type attributes
+
+typealias MAIsolated = @MainActor () -> ()
```

Comment:
Could you add a test to check that we also walk into the actual type and not just the attributes? Eg. the following should also report `Int`, I think.

```swift
typealias MAIsolated = @MainActor (Int) -> ()
```

Reply:
Updated. (Fortunately, it works without additional changes.)